### PR TITLE
Do not upload docs for PRs on Gitlab that are targeted towards main

### PR DESCRIPTION
**Pull request by kaushikcfd**
Here is the latest doc build of loopy that includes docs from kernel callables:

### CODE REVIEW

File: ci-support.sh [hunk 1]
```diff
@@ -434,7 +434,8 @@ build_docs()
 
 maybe_upload_docs()
 {
-  if test -n "${DOC_UPLOAD_KEY}" && test "$CI_DEFAULT_BRANCH" && test "$CI_COMMIT_REF_NAME" = "$CI_DEFAULT_BRANCH"; then
+  if test -n "${DOC_UPLOAD_KEY}" && test "$CI_DEFAULT_BRANCH" && \
+      test "$CI_COMMIT_BRANCH"  && test "$CI_COMMIT_BRANCH" = "$CI_DEFAULT_BRANCH"; then
```
Comment (kaushikcfd) [inline ci-support.sh:438]: Didn't test it, but if I understand the [gitlab docs](https://docs.gitlab.com/ee/ci/variables/predefined_variables.html#predefined-variables-reference) this should be fine.

**Comment by inducer** [comment]
> Here is the latest doc build of loopy that includes docs from kernel callables:

:rofl: Sorry. That was me. I was fixing the pytato doc build, which includes references to `loopy.TranslationUnit`. (https://github.com/inducer/pytato/pull/20/commits/78b9aec17674dd3967ba4d66ffc7e436c1255239) Rather than remove those references only to re-add them later, I simply uploaded kcv3 docs manually, so that the pytato doc build would succeed.

**Comment by inducer** [comment]
But thanks for having an eye on this!

**Comment by kaushikcfd** [comment]
:) Oops thanks! Closing this then.

**Pull request closed without merging**